docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -5,121 +5,45 @@ How to Prepare a Release
 
 In this guide, we'll cover:
 
-* preparing the code for a release
-* releasing the code on Github
-* updating the sha256
-* releasing the software on Anaconda
+* bumping the version for a release
+* releasing on Github
+* releasing on Anaconda
 * creating a new version of the documentation
 * building and releasing the Docker image
 
-Preparing The Code For Release
-------------------------------
+Bumping the Version
+-------------------
 
-These steps entail modifying files before a release is made.
-
-1. Pull the lastest code from whatever branch you want to release from.
```

Comment:
The commands for checking out the latest master should be excluded since the user should be familiar with their git environment at this point (forking or pulling). Less doc upkeep for us.